actioncable/app/assets/javascripts/action_cable/consumer.coffee

```diff
@@ -14,6 +14,17 @@
 #   App.appearance = App.cable.subscriptions.create "AppearanceChannel"
 #
 # For more details on how you'd configure an actual channel subscription, see ActionCable.Subscription.
+#
+#  To stop/disconnect from the server, call
+#
+#  App.cable.stop()
+#
+#  and to restart the connection:
+#
+#  App.cable.start()
+#
+#  Any channel subscriptions which existed prior to disconnecting will
+#  automatically rescubscribe.
```

Comment:
Non-standard indentation going on in the added docs. It goes, one for text, three for code.